README.md

```diff
@@ -152,12 +152,12 @@ the 2023 assessment model.
 | Percent Population Mobility, Moved From Within Same County in Past Year | acs5           | numeric   |                       |
 | Longitude                                                               | loc            | numeric   |                       |
 | Latitude                                                                | loc            | numeric   |                       |
+| Municipality Name                                                       | loc            | character |                       |
```

Comment:
This is just an artifact of removing the `var_name_model != "loc_tax_municipality_name"` conditional in `README.Rmd` above. I'm not sure why it changes the order of the feature in the table, but it represents the reverse of what happened in https://github.com/ccao-data/model-condo-avm/pull/4.